mlir/include/mlir/Dialect/MIOpen/MIOpen.h

```diff
@@ -161,6 +163,8 @@ class CoordTransformsBuilder {
 
   virtual int64_t paddingSign() const = 0;
 
+  llvm::SmallVector<SmallString<8>, 8> &nStartNames() { return startNames; }
```

Comment:
I don't think this needs an `n` on it

But it is a good method that's missing.

Reply:
rename to getStartName()

Reply:
Reminder on rename